src/app/secure-layout/secure-layout.component.css

```diff
@@ -40,13 +40,13 @@
 #main-container {
     display: flex;
     flex-direction: column;
-    background: url("../../assets/bg-sec-main.png");
+    /* background: url("../../assets/bg-sec-main.png");
     background-repeat: no-repeat;
-    background-size: auto;
+    background-size: auto; */
     min-height: calc(100vh - 50px);
     margin-top: 50px;
-    background-position-x: right;
-    background-position-y: top;
+    /* background-position-x: right;
+    background-position-y: top; */
```

Comment:
Either delete or document why is this commented out.

Reply:
@Skafec ?